rules-reviewed/hibernate/hibernate.windup.xml

```diff
@@ -30,6 +30,65 @@
                     </hint>
                 </iteration>
             </perform>
+        </rule>	
+	<!-- This rule may throw exception because some of the operations may not have it's condition part-->
+	<rule id="hibernate-02000">
+            <when>
+		<or>
+                <javaclass references="org.hibernate.criterion.Projections.count{*}" as="1">
+			<location>METHOD_CALL</location>
```

Comment:
This rule is overly complex. This is a good opportunity to use a parameter:

`<javaclass references="org.hibernate.criterion.Projections.{method}{*}">`

`<where param="method"><matches pattern="(count|rowCount|sum)" /></where>`

This would reduce this rule's verbosity by about 80%. Please update :)